components/02-elements/price/price.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="price {{ class }} {{{ attributes }}}">
+{{#if specialPrice }}
```

Comment:
It's better to add a variant with own template.

Reply:
done